practices/practice-one/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import { ReactNode, MouseEvent } from "react";
+import { NavLink } from "react-router-dom";
+import "./button.css";
+type ClickEvent = (event: MouseEvent) => void;
+
+interface ButtonProps {
```

Comment:
Disabled state of button?

Reply:
I don't understand this part, can you explain more?

Reply:
should have **disabled** prop